site/app/templates/admin/Extensions.twig

```diff
@@ -37,22 +37,22 @@
                 Student ID:
                 <input name="user_id" id="user_id" class="option-input" type="text">
             </label>
-            <label for="late_days">
+            <label for="late-days">
                 Give the Days of Extension:
                 <input name="late_days" id="late-days" class="option-input" type="text" onkeyup="clearDate()">
             </label>
-            <label for="late_calendar">
+            <label for="late-calendar">
                 Or Pick a New Due Date:
                 <input name="late_calendar" id="late-calendar" class="option-input" type="text" autocomplete="off" onchange="setLateDays()">
             </label>
             <a class="btn btn-primary" href="javascript:updateHomeworkExtension()">Submit</a>
         </fieldset>
         <fieldset>
-            <legend class="option-title">Multiple Student Entry Via CSV Upload</legend>
-            <label for="csv_upload">
+            <legend class="option-title" id="csv_upload_title">Multiple Student Entry Via CSV Upload</legend>
+            <label for="csv_upload" id="csv_upload_format">
                 Do not use column headers. CSV must be of the following form: student_id, gradeable_id, days_of_extension
             </label>
-            <input type="file" name="csv_upload" aria-labelledby="csv_upload_title csv_upload_format" onchange="updateHomeworkExtension()">
+            <input type="file" name="csv_upload" id="csv_upload" aria-labelledby="csv_upload_title csv_upload_format" onchange="updateHomeworkExtension()">
```

Comment:
Could you please update the `id` and `aria-labelledby` to use `kebab-case` here?

Reply:
done